tests/phpunit/compat/class-wc-stripe-subscription-renewal-test.php

```
@@ -378,6 +378,96 @@ public function test_renewal_authorization_required() {
+
+		// Assert: the renewal order itself is marked as failed.
+		$order = wc_get_order( $renewal_order->get_id() );
+		$this->assertSame( OrderStatus::FAILED, $order->get_status() );
+
+		// Assert: the parent subscription was put on hold to stop WC Subscriptions retries.
+		$this->assertSame( 'on-hold', $mock_subscription->get_status() );
+
+		// Clean up.
+		remove_filter( 'pre_http_request', $pre_http_request_callback );
+		WC_Subscriptions_Helpers::$wcs_get_subscriptions_for_renewal_order = null;
+	}
```

PHPMD (2.15.0)

[error] The method test_renewal_radar_blocked_puts_subscription_on_hold is not named in camelCase. (undefined)

(CamelCaseMethodName)